docs/dapps/oev-rewards/index.md

```diff
@@ -31,7 +31,11 @@ Use [this form](https://api3dao.typeform.com/to/FHhFIL41) to get in contact with
    ::: info 💡 Tip
    Alternatively, you can complete this step on your own using [API3 Market.](https://market.api3.org/)
    :::
-4. You [integrate](/dapps/integration/contract-integration) the OEV Rewards-enabled Api3ReaderProxyV1 contracts.
+4. You let us know an address where to receive the OEV proceeds.
```

Comment:
This answers one of the basic questions of the dApp, which is "How exactly do I get paid?"

Reply:
It's either OEV Rewards or auction proceeds. In this case, we're referring to OEV Rewards, which includes MEV revenue.